### PR TITLE
Mailing

### PR DESCRIPTION
**Pull request by valeriaDD**

### CODE REVIEW

File: resources/views/Emails/ContactUs.blade.php [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+<html lang="en">
+<body>
+    <h1>
+        User send an email
+    </h1>
+    <p>
+        user email : {{ $email }}
+    </p>
+    <p>
+        user name : {{ $name }}
```
Comment (egorbwork) [inline resources/views/Emails/ContactUs.blade.php:11]: Gender is not used here

File: app/Http/Controllers/ContactsController.php [hunk 2]
```diff
@@ -2,11 +2,41 @@
...
+        $data = $request->validated();
+        
+        \Log::debug('test',$data);
+
+        \Mail::send('Emails.ContactUs',
+                    [
+                        'email' => $data['email'],
+                        'name' => $data['name'],
+                        'gender' => $data['gender'],
+                        'messageText' => $data['messageText']
```
Comment (egorbwork) [inline app/Http/Controllers/ContactsController.php:29]: if you keep the name of input like messageText you should just use `$data` here.

**Review by egorbwork — CHANGES_REQUESTED** [review]
Looks nice, but still a bit incomplete.

File: resources/views/Emails/ContactUs.blade.php (hunk 1 above)
Comment (valeriaDD) [reply resources/views/Emails/ContactUs.blade.php:23]: fixed - added information about the gender as well

File: app/Http/Controllers/ContactsController.php (hunk 2 above)
Comment (valeriaDD) [reply app/Http/Controllers/ContactsController.php]: Fixed, accessed directly the data - without fracturing it

**Review by egorbwork — APPROVED** [review]
This part is ready. You can merge it after solving conflicts.

**Pull request merged**